python/ray/tune/trial_runner.py

```diff
@@ -163,8 +162,8 @@ def _serialize_and_write():
             search_alg.save_to_dir(self._checkpoint_dir, session_str=self._session_str)
 
         checkpoint_time_start = time.monotonic()
-        with out_of_band_serialize_dataset():
```

Comment:
why removing it? Is it causing you issues?

Reply:
No more problems :-)